### PR TITLE
Added controller to install script

### PR DESCRIPTION
**Pull request by Lallassu**

### CODE REVIEW

File: easy_install/script.sh [hunk 1]
```diff
@@ -18,6 +18,7 @@ OVERWRITE=${OVERWRITE=y}
 declare -A INSTALL_TBL
 INSTALL_TBL[41]="varnish-plus varnish-plus-vmods-extra varnish-plus-addon-ssl varnish-agent varnish-plus-ha varnish-broadcaster"
 INSTALL_TBL[60]="varnish-plus varnish-plus-vmods-extra varnish-plus-addon-ssl varnish-agent varnish-plus-ha varnish-broadcaster"
+INSTALL_TBL[controller]="varnish-controller-agent varnish-controller-brainz varnish-controller-api-gw varnish-controller-nats varnish-controller-cli"
```
Comment (gquintard) [inline easy_install/script.sh:21]: aren't we going to version this repo?

Comment (Lallassu) [reply easy_install/script.sh:21]: I have no idea. I guess that is @stone's area

Comment (stone) [reply easy_install/script.sh:21]: Don't know what this is about, sorry.

**Comment by gquintard** [comment]
now that the `controller` is out, I think we can look at this again.

only question from me is: what should we install by default? `agent`, `cli`, the server components, nothing?

**Comment by Lallassu** [comment]
> now that the `controller` is out, I think we can look at this again.
> 
> only question from me is: what should we install by default? `agent`, `cli`, the server components, nothing?

I have no idea what the script is really about, it was you that asked me to add controller to it long time ago :grin: 

Where is it supposed to be run? Is it to be used for simple installation for test? or customers?

Could divide between controller-server (brainz, cli, apigw, nats) vs. varnish-server (agent, nats) perhaps?

**Comment by gquintard** [comment]
@aondio , just for the thrill of tagging people :-)

**Comment by gquintard** [comment]
timing this one out

**Pull request closed without merging**